crates/harp/src/environment.rs

```diff
@@ -91,13 +91,13 @@ impl Environment {
         std::iter::successors(Some(self.clone()), |p| p.parent())
     }
 
-    pub fn bind(&self, name: RSymbol, value: impl Into<SEXP>) {
+    pub fn bind(&self, name: RSymbol, value: SEXP) {
```

Comment:
hmm or maybe make it take an `&RObject`?

Reply:
Done in https://github.com/posit-dev/ark/pull/641/commits/94eba522c020eb6c75a91900382af85b9fbafee5